pyuvsim/data/test_config/28mEWbl_10time_10chan.yaml

```diff
@@ -0,0 +1,5 @@
+Nants: 2
```

Comment:
Why do we want Nants here? I would think that should be dictated by the number of rows in the layout csv. That way you can use different layouts with the same telescope file.

Reply:
I wrote it originally thinking it would use Nants as an extra constraint. I'll switch to using the number of rows instead.

I just noticed though, while getting my nosetests to work on Oscar, that the config files require the full paths to the telescope parameters and layout files. At the moment, they're set to the paths on my laptop. Perhaps I can have it default to checking the path relative to the location of the config file?